### ISSUE TITLE
Correctly handle UUIDs in Model::getRelated()

### ISSUE THREAD

**Issue by arboc** [contao/core#8850]
After changeing Model.php Line 627 to
`$strField = $arrRelation['table'] . '.' . \Database::quoteIdentifier($arrRelation['field']);`

the following condition Line 630 doesn't work any more
`if ($strField == 'tl_files.uuid')`

because the $strField is quoted.

**Comment by ausi** [comment]
Should probably changed to:

```php
if ($arrRelation['table'] === 'tl_files' && $arrRelation['field'] === 'uuid')
```

**Comment by arboc** [comment]
Or: 
```php
if ($strField == 'tl_files.`uuid`')
```

**Comment by fritzmg** [comment]
This issue is still assigned to the `3.5.33` milestone, however it hasn't actually been fixed in Contao `3.5.33`.

**Comment by leofeyer** [comment]
Nope, wrong milestone.

**Comment by leofeyer** [comment]
Fixed in a0880b971b850f20561dc0b2b2769c4578d79c3d.
